lib/api/src/tests/stories.test.js

```diff
@@ -144,6 +153,14 @@ describe('stories API', () => {
         name: '2',
         parameters,
       });
+
+      expect(storedStoriesHash['custom-id--1']).toMatchObject({
```

Comment:
Can we add the expected state of `storedStoriesHash['b-e']` to this test? If nothing else it would help me understand the above code